.github/workflows/kics.yml

```
@@ -0,0 +1,40 @@
+name: kics
+
+on:
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - "docs/**"
+      - "*.md"
+      - .github/workflows/docs.yml
+      - .pre-commit-config.yaml
+  push:
+    branches: [master]
+    paths-ignore:
+      - "docs/**"
+      - "*.md"
+      - .github/workflows/docs.yml
+      - .pre-commit-config.yaml
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  kics:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: run kics Scan
+        uses: checkmarx/kics-github-action@530ac1f8efe6202b0f12c9a6e952597ae707b755 # v2.1.2
+        with:
+          path: 'playbooks,roles'
+          ignore_on_exit: results
+          output_path: report-dir/
+          output_formats: 'sarif'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          enable_jobs_summary: true
```

```suggestion
          enable_jobs_summary: true
          enable_comments: false
```

we agreed on that right?

---

enable_comments defaults to false

---

https://github.com/Checkmarx/kics-github-action?tab=readme-ov-file#inputs

I was assuming the first column was `default` 🤦🏻

---

Yeah not very intuitive - also the column with default values is hidden a little :D